cSpell.json

```diff
@@ -0,0 +1,108 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
+    "Alloc",
+    "analyse",
```

Comment:
ℹ️ Typically I standardize on `en-US` because that's the cSpell default. In this case, it seems `analyse` is pretty "baked in" to the docs so I chose to add it instead. Let me know if you'd like me to adjust to `analyze` and I will.